models/upload.go

```diff
@@ -130,6 +132,12 @@ func DeleteUploads(uploads ...*Upload) (err error) {
 		}
 
 		if err := os.Remove(localPath); err != nil {
+			i := 0
+			for i < 5 && strings.Contains(err.Error(), "The process cannot access the file because it is being used by another process") {
```

Comment:
I imagine this string will be affected by the localization setting of Windows. Are there error codes available?

Reply:
If you can find one... I couldn't

Reply:
Maybe just retry regardless of error message?

(Does golang generally really not provide error codes, only messages?)